mobile/native/src/api.rs

```diff
@@ -136,6 +137,9 @@ pub fn subscribe(stream: StreamSink<event::api::Event>) {
 }
 
 pub fn run(app_dir: String) -> Result<()> {
+    let db_path = format!("sqlite://{app_dir}/trader.sqlite");
```

Comment:
🔧 Wouldn't it be better to simply pass the `app_dir` to `db::init_db` and deriving the `database_url` within the function, instead of outside? It feels out of place to do that on the outside, as it implies knowledge about the inner workings of `db::init_db`

set the sqlite specific database_url within the

Reply:
Ah, got it, you would only pass in the `app_dir` and add the `taker.sqlite` inside. Can do :)